packages/conf-ncurses/conf-ncurses.1/opam

```diff
@@ -7,6 +7,9 @@ license: "MIT"
 build: [
   ["pkg-config" "ncurses"] { os != "darwin"}
 ]
+depends: [
+  "conf-pkg-config" { os != "darwin"}
```

Comment:
I did the audit. The dependency is missing in almost every `conf-*` that would require it. I did a quick patch, but I was afraid of the big change. I was planning doing this slowly while auditing `depext` which are not shared in a `conf-*`.